installer-tests/Vagrantfile

```diff
@@ -60,6 +65,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  # Use qemu driver for libvirt. See README.md for rationale.
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.driver = 'qemu'
```

Comment:
On the other hand, maybe this thing can live with a static amount of RAM and number of CPUs, since all we're testing is the install script!